### ISSUE TITLE
Jump to Version - Appears to be broken for distributions with different authors

### ISSUE THREAD

**Issue by mrdvt92** [metacpan/metacpan-web#2201]
"Jump to Version" link for distribution where the authors change appears to not work.

And Example is CGI try to Jump to 3.63 which is the supported version in RedHat EL7.

**Comment by Grinnz** [comment]
Works for me - it goes to https://metacpan.org/release/MARKSTOS/CGI.pm-3.63.

**Comment by mrdvt92** [comment]
I get a 404 link to https://metacpan.org/pod/release/MARKSTOS/CGI.pm-3.63/lib/CGI.pod

Steps to Reproduce
Chrome Version 75.0.3770.142 (Official Build) (64-bit) on Windows
https://metacpan.org/
Search CGI => https://metacpan.org/search?size=200&q=CGI
Click CGI link => https://metacpan.org/pod/distribution/CGI/lib/CGI.pod
Click Jump to 3.63 => https://metacpan.org/pod/release/MARKSTOS/CGI.pm-3.63/lib/CGI.pod (HTTP 404)

Work around
Search CGI => https://metacpan.org/search?size=200&q=CGI
Click CGI link => https://metacpan.org/pod/distribution/CGI/lib/CGI.pod
{workaround} Click on the link "CGI-4.44" => https://metacpan.org/release/CGI
Click Jump to 3.63 => https://metacpan.org/release/MARKSTOS/CGI.pm-3.63

**Comment by mrdvt92** [comment]
I wonder if #1257 broke this?

**Comment by Grinnz** [comment]
Try going from https://metacpan.org/pod/CGI. I think the problem is you are on the page for a specific file path, and that file doesn't exist in that version (its docs are in CGI.pm)

**Comment by mrdvt92** [comment]
Correct.  So, if the jump_to distribution does not have the same file as the jump_from distribution, then the UI needs to go to a default page in the distribution.

something like this.
```
If (exists $jump_to->page($current_page->pathname)) {
  redirect($jump_to->page($current_page->pathame));
} else {
  redirect($jump_to->page_default);
}
```

**Comment by mickeyn** [comment]
The `CGI.pod` file your path is looking for doesn't exist in the version you're trying to jump to.
see https://st.aticpan.org/source/MARKSTOS/CGI.pm-3.63/lib/ vs https://st.aticpan.org/source/LEEJO/CGI-4.44/lib/

Sure, we should do better filtering some generated paths but generally it looks like a low-prio issue.
